platform-test/base/tailrec.oz

```diff
@@ -0,0 +1,79 @@
+functor
+export
+   Return
+define
+   Return='tailrec'([
+		     tailProc(proc {$}
+				 proc {Rec I}
+				    if I == 0 then
+				       {Rec I-1}
+				    else
+				       skip
+				    end
+				 end
+			      in
+				 {Rec 100000}
```

Comment:
Could you extract this number in a constant and add a comment that this number is large enough to only work if calls are tail calls?